index.js

```diff
@@ -274,10 +275,29 @@ function startSyncWithRetry(watcherArray, maximumNumberOfAttempts, pauseSeconds)
 }
 
 app.listen(config.net.port, () => {
-  startSyncWithRetry(watchers, config.start.attempts, config.start.pauseSeconds);
-
-  debug('finder ' + config.version.major + '.' + config.version.minor + '.' +
-    config.version.bug + ' with api version ' + config.version.api +
-    ' waiting for requests on port ' + config.net.port);
+  //delete existing index and create new index with spatial mapping 
+  esclient.indices.exists({ index: config.elasticsearch.index })
+  .then(function (resp) {
+    if (resp) { //if config.elasticsearch.index already exists
+      //delete existing index
+      return esclient.indices.delete({ index: config.elasticsearch.index });
+    }
+  }).then(function (resp) {
+    //(re)create index
+    return esclient.indices.create({ index: config.elasticsearch.index });
+  }).then(function (resp) {
+    //add spatial mapping
+    return esclient.indices.putMapping({ index: config.elasticsearch.index, type: "compendia", body: mapping });
+  }).then(function (resp) {
+    debug("Created elasticsearch index and mapping");
+    // start listening when the elasticsearch index was created
+    startSyncWithRetry(watchers, config.start.attempts, config.start.pauseSeconds);
+
+    debug('finder ' + config.version.major + '.' + config.version.minor + '.' +
```

Comment:
Change this to use the version from `package.json`, see https://github.com/o2r-project/o2r-contentbutler/blob/master/config/config.js#L23 and https://github.com/o2r-project/o2r-contentbutler/blob/master/index.js#L132 (and no more string concatenation).